content/docs/command-reference/checkout.md

```diff
@@ -207,8 +202,29 @@ $ md5 model.pkl
 MD5 (model.pkl) = 98af33933679a75c2a51b953d3ab50aa
 ```
 
-What happened is that DVC went through the DVC-files and adjusted the current
-set of <abbr>output</abbr> files to match the `outs` in them.
+DVC went through `dvc.lock` and adjusted the current set of <abbr>outputs</abbr>
+to match the `outs` in it.
+
+## Example: Specific files or directories
+
+`dvc checkout` only affects the tracked data corresponding to any given
+`targets`:
+
+```dvc
+$ git checkout master
+$ dvc checkout            # Start with latest version of everything.
+
+$ git checkout baseline-experiment -- dvc.lock
+$ dvc checkout model.pkl  # Get previous model file only.
+```
+
+Note that `dvc checkout` supports granular targeting of files inside directories
+that are tracked as a whole. For example, the `featurize` stage has a directory
+output (`data/features`) and we can do:
+
+```dvc
+$ dvc checkout data/features/test.pkl
+```
```

Comment:
This actually doesn't seem to work... Opened iterative/dvc/issues/4202

Reply:
@jorgeorpinel, which one does not work? `checkout` or the `list`?

Reply:
Good point sorry for confusion. I think I meant both. Let me check again now that it's been solved... ⏳

Reply:
Both work now so... Moving on! Thanks